integration_tests/test_demographics.py

```diff
@@ -0,0 +1,45 @@
+import pandas as pd
+
+from vivarium_census_prl_synth_pop.constants import metadata
+
+
+# TODO: Broader test coverage
+def test_initial_population_size(sim, tracked_live_populations):
+    pop_size = sim.configuration.population.population_size
+    pop = tracked_live_populations[0]
+
+    assert pop.index.size == pop_size
+    assert (pop["alive"] == "dead").sum() == 0
```

Comment:
This population is alive by definition so if you want to check for this you should use `populations` instead of `tracked_live_populations`.